target/ppc/translate.c

```diff
@@ -1273,6 +1273,12 @@ typedef struct opcode_t {
         gen_priv_exception(ctx, POWERPC_EXCP_PRIV_OPC); return; \
     } while (0)
 
+#define GEN_PRIV2(ret)                                          \
```

Comment:
Hum... maybe we should make `GEN_PRIV` receive `ctx` as an argument (as the others `REQUIRE_{FPU,VECTOR,VSX,INSNS_FLAGS,etc}`), and then make it variadic, e.g.
```c
#define GEN_PRIV(CTX, ...)  \
    do {                                                \
        gen_priv_exception(CTX, POWERPC_EXCP_PRIV_OPC); \
        return __VA_ARGS__ ;                            \
    } while (0)
```
It still a bit clunky tho, it might be better to just call `gen_priv_exception` directly...

Reply:
I wanted to avoid changing the other 70 GEN_PRIV uses.
I guess I'll go with the 'just call `gen_priv_exception`' option for now, because currently `do_tlbie()` is the only place that also needs to return a value.
OTOH, there is a hidden use of the local variable ctx by GEN_PRIV, so, later, changing it to always explicitly receive ctx seems a good idea.

Reply:
Removed GEN_PRIV2 and replaced it by calling `gen_priv_exception` directly.